event-management-backend/src/test/java/ro/msg/event_management/SaveEventIntegrationTest.java

```diff
@@ -65,27 +81,21 @@ void testSaveEvent() {
                 .ticketsPerUser(10)
                 .highlighted(false)
                 .maxPeople(30)
-                .startDate(LocalDate.now())
-                .endDate(LocalDate.now())
-                .startHour(LocalTime.now())
-                .endHour(LocalTime.now().plusHours(3))
+                .startDate(LocalDate.parse("2020-11-11"))
+                .endDate(LocalDate.parse("2020-11-15"))
+                .startHour(LocalTime.parse("18:00"))
+                .endHour(LocalTime.parse("20:00"))
```

Comment:
I do not see a reason to change to specific dates.

Reply:
Switched back to dates relative to the current time.